tsconfig.test.json

```
@@ -0,0 +1,8 @@
+{
+	"extends": "./tsconfig.json",
+	"include": [
+		"./typings/",
+		"./packages/*/src/augmentation.ts",
+		"./external/*/packages/*/src/augmentation.ts"
```

```suggestion
		"./external/ckeditor5-internal/packages/*/src/**/*.ts",
		"./external/collaboration-features/packages/*/src/**/*.ts",
```

To match https://github.com/ckeditor/ckeditor5/blob/0e5ddd74e5684897d2d2c87a0e65150c287433d4/tsconfig.docs.json#L5-L6. People can have more than two repositories in `external/` and it should. not be loaded.

Btw, we use spaces in all JSON files.

---

I updated it to:  
 

```
"./external/ckeditor5-internal/packages/*/src/augmentation.ts",
"./external/collaboration-features/packages/*/src/augmentation.ts"
```